index.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const models = require('./models')
+const app = express()
+const Op = require('sequelize')
+
+
+app.get('/teams', async (request, response) => {
+  const teams = await models.teams.findAll()
+  response.send(teams)
+})
+
+app.get('/teams/:id', async (request, response) => {
+  const { id } = request.params.id
+
+  const matchingTeam = await models.teams.findAll({
+    where: {
+      [Op.or]: [{ id: id }, { abbreviation: id.toUpperCase() }]
+    }
+  })
+
+  if (matchingTeam.length) {
+    response.send(matchingTeam)
+  } else {
+    response.sendStatus(418)
+  }
+})
+
+app.post('/teams', bodyParser.json(), async (request, response) => {
+  const { id, location, mascot, abbreviation, conference, division } = request.body
```

Comment:
We should let the database take care of `id` for us rather than relying on a user provided value and messing up the auto increment on the field.